apps/token-manager/buidler.config.js

```diff
@@ -0,0 +1,39 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+const hooks = require('./scripts/buidler-hooks')
+
+usePlugin('@aragon/buidler-aragon')
+usePlugin('@nomiclabs/buidler-solhint')
+usePlugin('buidler-gas-reporter')
+usePlugin('solidity-coverage')
+
+module.exports = {
+  defaultNetwork: 'localhost',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545',
+      accounts: {
+        mnemonic: "explain tackle mirror kit van hammer degree position ginger unfair soup bonus"
```

Comment:
I wonder if this is something we would want to expose as part of the buidler plugin, e.g. as a default mnemonic?

Does it matter if we use this (e.g. to get the correct contract deployments)?

Reply:
As mention above it's not necessary actually.